build-system/tasks/size.js

```diff
@@ -102,6 +100,9 @@ function normalizeRows(rows) {
   // normalize alp.js
   normalizeRow(rows, 'alp.js', 'alp.max.js', true);
 
+  // normalize amp-shadow.js
+  normalizeRow(rows, 'shadow-v0.js', 'amp-shadow.js', true);
```

Comment:
TODO: rename this with .max if https://github.com/ampproject/amphtml/pull/3752 is merged